sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -1288,7 +1288,7 @@ private async Task RenewOwnershipAsync(CancellationToken cancellationToken)
         ///
         /// <returns>The running task that is currently receiving and processing events in the context of the specified partition.</returns>
         ///
-        private Task RunPartitionProcessingAsync(string partitionId,
+        internal virtual Task RunPartitionProcessingAsync(string partitionId,
                                                  EventPosition startingPosition,
```

Comment:
nit:  If you would be so kind, could you line up the second and third parameter with the `partitionId` declaration?